korman/properties/modifiers/render.py

```diff
@@ -73,6 +74,10 @@ def draw_opaque(self):
     def draw_framebuf(self):
         return self.render_level == "FRAMEBUF"
 
+    @property
+    def draw_late(self):
+        return self.render_level == "LATE"
```

Comment:
This property needs a stub on the `PlasmaModifierProperties` base class:
```python
    def draw_late(self):
        return False
```

to ensure that all modifiers will return a value for this property.